tests/shareRecipientTest.js

```diff
@@ -1,16 +1,15 @@
-describe('Main: Currently testing share recipient,', function () {
+fdescribe('Main: Currently testing share recipient,', function () {
   // CURRENT TIME
-  var timeRightNow = Math.random().toString(36).substr(2, 9)
   var OwnCloud = require('../src/owncloud')
   var config = require('./config/config.json')
 
   // LIBRARY INSTANCE
   var oc
 
   // TESTING CONFIGS
-  var testUserPassword = 'password'
-  var testUser = 'testUser' + timeRightNow
-  var testGroup = 'testGroup' + timeRightNow
+  var testUserPassword = config.testUserPassword
+  var testUser = config.testUser
+  var testGroup = config.testGroup
 
   beforeEach(function (done) {
     oc = new OwnCloud({
```

Comment:
While you are on this file change all the `var`  to `const`

Reply:
fixed